cpp/src/slice2cpp/Gen.cpp

```diff
@@ -1111,9 +1111,14 @@ Slice::Gen::ForwardDeclVisitor::visitEnum(const EnumPtr& p)
         H << "class ";
     }
     H << getDeprecatedAttribute(p) << fixKwd(p->name());
-    if (!unscoped && p->maxValue() <= 0xFF)
+    if (!unscoped)
```

Comment:
Unfortunately we have no test for unscoped: we should either test it or remove it.